<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncodingForm</class>
 <widget class="QWidget" name="EncodingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置编码格式</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>无法打开文件, 正确的编码格式.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_0">
       <property name="text">
        <string>编码格式</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmb_encoding">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>自动识别</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UTF-8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>GBK</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BIG5</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pbtn_ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pbtn_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
